PowerEditor/installer/APIs/nsis.xml

```diff
@@ -98,22 +107,27 @@
 		<KeyWord name="GetDLLVersion" />
 		<KeyWord name="GetDLLVersionLocal" />
 		<KeyWord name="GetErrorLevel" />
-		<KeyWord name="GetExeName" />
-		<KeyWord name="GetExePath" />

```

Comment:
Why are both APIs removed?

Reply:
They do not even exist in the first CVS2SVN check-in 20 years ago: https://sourceforge.net/p/nsis/code/625/tree/NSIS/trunk/Source/tokens.cpp There are some macros with those names but the syntax would be ${GetExeName} and they are not a part of the base instructions.

$EXEPATH and $EXEFILE variables have been around for 15 years.